packages/ra-ui-materialui/src/field/ImageField.spec.js

```diff
@@ -29,9 +29,9 @@ describe('<ImageField />', () => {
         );
 
         const img = getByRole('img');
-        expect(img.prop('src')).toEqual('http://foo.com/bar.jpg');
-        expect(img.prop('alt')).toEqual('Hello world!');
-        expect(img.prop('title')).toEqual('Hello world!');
+        expect(img.src).toEqual('http://foo.com/bar.jpg');
```

Comment:
That works too :+1:

Reply:
How can I run these test files individually on my system? I am trying to run using jest but it giving syntax error because of `testing-library`.

Reply:
Can you paste the syntax error here?

Reply:
![TestFail](https://user-images.githubusercontent.com/28812464/63504855-0cf3a500-c4f0-11e9-8d7c-af3e2f3b7aee.png)